textproc/py-stanfordnlp/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                                         1.0
+PortGroup           python                         1.0
+PortGroup           java                           1.0
+PortGroup           github                         1.0
+
+name                py-stanfordnlp
+version             0.2.0
+revision            0
+github.setup        stanfordnlp stanfordnlp ${version} v
+
+categories          textproc java
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         A Python NLP Library for Many Human Languages
+long_description    The Stanford NLP Group's official Python NLP \
+                    library. It contains packages for running our \
+                    latest fully neural pipeline from the CoNLL 2018 \
+                    Shared Task and for accessing the Java Stanford \
+                    CoreNLP server.
+
+homepage            https://stanfordnlp.github.io/stanfordnlp/
+
+github.livecheck.regex  {([0-9.]+)}
+
+checksums           rmd160  95e41e706e48120a6c3923c987404538b3da37af \
+                    sha256  549c6993d0088583c35eab33f8c28e5dfdfe7c5153370d4c01bc26de56cdc46e \
+                    size    130852
+
+# Required java version
+java.version        11+
+# JDK port to install if required java not found
+java.fallback       openjdk13
+
+# Support python versions
+python.versions        27 37
+python.default_version 37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+    depends_run-append \
```

Comment:
from their [```setup.py```](https://github.com/stanfordnlp/stanfordnlp/blob/d270c0b40c8a5b975fe77a70427adeb07e8ea4f0/setup.py#L69) it appears your missing dependencies

Reply:
I don’t see that: https://github.com/stanfordnlp/stanfordnlp/blob/master/setup.py

Also, it builds successfully, so…